src/stock-time-series/StockTimeSeries.ts

```diff
@@ -225,4 +228,22 @@ export class StockTimeSeries extends Category {
       );
     }
   }
+
+  async quote(quoteDTO: QuoteDTO): Promise<QuoteResponse> {
```

Comment:
please add the unit tests for this method, the tests should be something similar to the `search `method unit tests